src/components/component-playground.jsx

```diff
@@ -198,11 +206,17 @@ module.exports = React.createClass({
   },
 
   _renderPreview: function() {
-    return <div ref="previewContainer"
-                key="previewContainer"
-                className={this._getPreviewClasses()}>
-      {this.loadChild('preview')}
-    </div>
+    store.dispatch(changeFixture(this.state.fixtureContents.reduxStore));
+
+    return <Provider key="provider" ref="provider" store={store}>
```

Comment:
oh man... why not usign `children` here?

```javascript
provider: () => {
  return {
    component: Provider,
    key: 'provider',
    store: store,
    children: this.loadChild('preview')
  }
}
```

_PS: That's the beauty of `component-tree`_